docs/core/compatibility/networking/6.0/webrequest-deprecated.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Breaking change: WebRequest, WebClient, and ServicePoint are obsolete"
+description: Learn about the breaking change in .NET 6.0 where WebRequest, WebClient, and ServicePoint are deprecated in favor of HttpClient.
+ms.date: 04/26/2021
+---
+# WebRequest, WebClient, and ServicePoint are obsolete
+
+<xref:System.Net.WebRequest>, <xref:System.Net.WebClient>, and <xref:System.Net.ServicePoint> classes are marked as obsolete and generate a `SYSLIB0014` warning at compile time.
+
+## Version introduced
+
+6.0
+
+## Change description
+
+<xref:System.Net.WebRequest>, <xref:System.Net.WebClient>, and <xref:System.Net.ServicePoint> classes were added to .NET Core in version 2.0 for backward compatibility. However, they introduced several runtime breaking changes, for example, `WebRequest.GetRequestStream` allocates memory for the whole response, and `WebClient.CancelAsync` doesn't always cancel immediately.
+
+Starting in .NET 6, the <xref:System.Net.WebRequest>, <xref:System.Net.WebClient>, and <xref:System.Net.ServicePoint> classes are deprecated. The classes are still available, but they're not recommended for new development. To reduce the number of analyzer warnings, only construction methods are decorated with the <xref:System.ObsoleteAttribute> attribute.
+
+## Recommended action
+
+Use the <xref:System.Net.Http.HttpClient?displayProperty=fullName> class instead.
+
+## Affected APIs
+
+- <xref:System.Net.WebRequest>
+- <xref:System.Net.HttpWebRequest>
+- <xref:System.Net.FtpWebRequest>
+- <xref:System.Net.WebClient>
+- <xref:System.Net.ServicePoint>
+
+<!--
+
+### Affected APIs
+
+- `T:System.Net.WebRequest`
+- `T:System.Net.HttpWebRequest`
+- `T:System.Net.FtpWebRequest`
+- `T:System.Net.WebClient`
+- `T:System.Net.ServicePoint`
+
+### Category
+
+Networking
```

Comment:
Core libraries feels like a valid category too, but how the docs are organized doesn't allow multiple categories. Are there any criteria to prefer one over the other?

Reply:
I think "Core .NET libraries" is for anything that doesn't fit in one of the other categories. So Networking is fine here.